src/App.tsx

```diff
@@ -1,12 +1,19 @@
-import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import todos from './api/todos';
+import users from './api/users';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoItem } from './types/TodoItem';
+
+const prepareTodos: TodoItem[] = todos.map(todo => ({
```

Comment:
З дієслова починаються назви функцій
```suggestion
const preparedTodos: TodoItem[] = todos.map(todo => ({
```